### PR TITLE
Added the contributors at the bottom of readme file

### PR DESCRIPTION
**Pull request by srisatya12**
@yashlamba @yashugarg  Sir I added the contributors at the bottom of the readme file i.e., #182  . Please review it and let me know .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,13 @@ Some pre-requisites:
 
 
 All further guidelines can be found [here](https://cod-ed.github.io/simulate/contributing).
+
+
+
+## Our Contributors  !! ✨
+### Thanks go to these wonderfull peoples: ✨
```
Comment (yashugarg) [inline README.md]: Remove this line (37) and the emoticon from the line 36

**Review by yashugarg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (srisatya12) [reply README.md]: Ok sure Sir.

**Comment by srisatya12** [comment]
Can you please check the updated version of readme file @yashugarg or @yashlamba Sir?

**Comment by srisatya12** [comment]
@yashlamba @yashugarg Can you please check it now I made the requested changes in the pr

**Comment by srisatya12** [comment]
@apoorvdwi Can you please check this one?

**Pull request merged**